plugins/modules/web_infrastructure/bugzilla.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Chuang Cao <chcao@redhat.com>
+# Atlassian open-source approval reference OSR-76.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: bugzilla
+version_added: "1.0.0"
```

Comment:
```suggestion
version_added: "1.2.0"
```
1.0.0 has already been released, and for 1.1.0 (to be released later today) it's probably too late.

Reply:
Ack